NUITizenGallery/res/layout/ShadowFrameTest1Page.xaml

```diff
@@ -0,0 +1,672 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<b:View x:Class="NUITizenGallery.ShadowFrameTest1Page"
+  xmlns="http://tizen.org/Tizen.NUI/2018/XAML"
+  xmlns:b="clr-namespace:Tizen.NUI.BaseComponents;assembly=Tizen.NUI"
+  xmlns:nui="clr-namespace:Tizen.NUI.Components;assembly=Tizen.NUI.Components"
```

Comment:
hope we use same namespace,
b->base
nui->comp

Reply:
I check up that we don't need to setup base:View or comp:Slider.